<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StoreManagerPage</class>
 <widget class="QMainWindow" name="StoreManagerPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>911</width>
      <height>591</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="storeManagerHomePage">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>STORE MANAGER PAGE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dailySalesReportsPushBtn">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>110</y>
        <width>231</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Go to Daily Sales Reports</string>
      </property>
     </widget>
     <widget class="QPushButton" name="totalPurchasesPushBtn">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>150</y>
        <width>231</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Go to Total Purchases</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rebatePushBtn">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>200</y>
        <width>231</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Go to Rebate and Expiration Page</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="rebateAndExpPage">
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>REBATE AND EXPIRATION PAGE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rebateBackBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>520</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QTableView" name="rebateAndExpTableView">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>70</y>
        <width>661</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="displayRebatePushBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Display Rebate</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>170</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Expiration Month:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="expMonthComboBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="expMonthEnterBtn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>240</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="dailySalesReportsPage">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>DAILY SALES REPORTS PAGE</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>711</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="dailySalesBackBtn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>530</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter a date:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dateEnterBtn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>181</x>
        <y>461</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Total Revenue (7.75%): </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="totalRevenueLineEdit">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>460</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="displayByExecBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Display by Executive</string>
      </property>
     </widget>
     <widget class="QPushButton" name="displayByRegBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Display by Regular</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>183</x>
        <y>491</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of Executive Members: </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="execMembersLineEdit">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>491</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>520</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of Regular Members: </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="regMembersLineEdit">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>520</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="dateComboBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="displayAllInfoBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>DISPLAY ALL</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="totalPurchasesPage">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TOTAL PURCHASES PAGE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="totalPurchasesBackBtn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>520</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QTableView" name="totalPurchasesTableView">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>531</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>500</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Grand Total (7.75%)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="grandTotalLineEdit">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>500</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="displayByItemBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Display by Item</string>
      </property>
     </widget>
     <widget class="QPushButton" name="displayByAllMembersBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Display by all Members</string>
      </property>
     </widget>
     <widget class="QPushButton" name="enterItemPushBtn">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>120</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Item</string>
      </property>
     </widget>
     <widget class="QPushButton" name="enterNamePushBtn">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>340</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter name</string>
      </property>
     </widget>
     <widget class="QComboBox" name="itemComboBox">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>70</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="nameComoboBox">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>300</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>40</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Item name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>270</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Member name:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>903</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
